src/main/java/task/Task.java

```diff
@@ -0,0 +1,43 @@
+package task;
+
+public class Task {
+    private int hasCompletedInt;
+    private String name;
+
+    /**
+     * Default Constructor for Task
+     *
+     * @param name Name of Task
+     */
+    public Task(String name) {
+        this.hasCompletedInt = 0;
+        this.name = name;
+    }
+
+    /**
+     * Alternative Constructor for Task
+     *
+     * @param name         Name of Task
+     * @param hasCompletedInt boolean to determine whether task is completed
+     */
+    public Task(String name, int hasCompletedInt) {
+        this.hasCompletedInt = hasCompletedInt;
+        this.name = name;
+    }
```

Comment:
Is there a reason you decided to include this alternative constructor for `Task` and its sub-classes? In my implementation I only included a single constructor, and used the `markAsDone` method when needed.

Reply:
I used this alternative constructor for reading the tasks from the file to the array list such that regardless whether the task has been marked as done the initialisation step will stay the same for better readability